### ISSUE TITLE
sidekiq 6.1.0 (master) and ArgumentError: wrong number of arguments (given 2, expected 1)

### ISSUE THREAD

**Issue by acarpe** [scoutapp/scout_apm_ruby#338]
just deployed latest version of our app with the sidekiq 6.1.0 (from master) and I started receiving this the sidekiq starts:

```
Jun 23 14:18:33 app[web] INFO pid=78 tid=gnrzs1252 INFO: Booting Sidekiq 6.1.0 with redis options {:url=>"redis://h:REDACTED@REDACTED:REDACTED"}
Jun 23 14:18:33 app[web] WARN pid=78 tid=gnrzs1252 WARN: ArgumentError: wrong number of arguments (given 2, expected 1)
Jun 23 14:18:33 app[web] WARN pid=78 tid=gnrzs1252 WARN: /app/vendor/bundle/ruby/2.6.0/gems/scout_apm-2.6.7/lib/scout_apm/background_job_integrations/sidekiq.rb:43:in `initialize_with_scout'
Jun 23 14:18:33 app[web] info /app/vendor/bundle/ruby/2.6.0/bundler/gems/sidekiq-ab6f16523709/lib/sidekiq/manager.rb:38:in `new'
Jun 23 14:18:33 app[web] info /app/vendor/bundle/ruby/2.6.0/bundler/gems/sidekiq-ab6f16523709/lib/sidekiq/manager.rb:38:in `block in initialize'
```

**Comment by dlanderson** [comment]
Thanks for the report @acarpe. We're looking into it.

**Comment by acarpe** [comment]
For now I removed scout gem because the errors prevents the workers to start

**Comment by dlanderson** [comment]
If you want to continue using Scout for your web apps, you can set the `enable_background_jobs` config param to avoid monitoring/instrumentation of Sidekiq. https://docs.scoutapm.com/#enable_background_jobs

We'll follow up soon with a proper fix.

**Comment by cschneid** [comment]
@acarpe - This is released in 2.6.8, can you verify it works for you?

**Comment by acarpe** [comment]
It works perfectly now, thank you!
